en/flight_modes/follow_me.md

```diff
@@ -8,16 +8,22 @@ author: Jimmy Johnson
 
 *Follow Me* mode allows a multicopter to autonomously follow and track another system that is broadcasting its position using the [FOLLOW_TARGET](https://mavlink.io/en/messages/common.html#FOLLOW_TARGET) MAVLink message.
 
-The vehicle will automatically yaw to face and follow the target from a specified [relative position](#NAV_FT_FS), [horizontal separation](#NAV_FT_DST) and [height](#NAV_MIN_FT_HT) above the home position.
-By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home/arming position.
-While in this mode no user input is required.
+![Follow-me Concept](../../assets/flight_modes/followme_concept.png)
 
-:::tip
-PX4 currently ignores the altitude of/from the target and follows at a constant altitude above home.
-This limitation is because altitude sources from GPS on ground stations are often inaccurate.
-:::
+The vehicle will automatically yaw to face and follow the target from a specified [relative angle](#NAV_FT_FS), [distance](#NAV_FT_DST) and [height](#NAV_FT_HT) and altitude, depending on the [altitude control mode](#NAV_FT_ALT_M)).
+
+By default it will follow from directly behind the target at a distance of 8 meters, and a height of 8 meters above the home(arming) position.
+
+User has an option to control the follow angle, height and distance by :
+- Follow Angle : Roll input. Left and right command will be as if you were controlling the drone's roll while facing it, i.e. Left command moves drone around counter clockwise
+- Follow Height : Throttle input. Center the stick to keep follow height constant, increase or decrease to adjust height
+- Follow Distance : Pitch input. Pushing pitch stick up increases the follow distance, pulling it down decreases the distance.

```

Comment:
This only works on MC, so we should probably use MC position mode terminology or a diagram - such as http://docs.px4.io/master/en/getting_started/flight_modes.html#position-mode-mc
i.e. to me the roll input is the left-right input :-)

```suggestion
User has an option to control the follow angle, height and distance by :
- _Follow Angle_ is controlled using the `left-right` input ("Roll"). The movement is from the user's perspective, so if you face the drone and move the stick left, it will move to your left. From above if you move the stick left the drone will move counter-clockwise.
- _Follow Height_ is controlled using the `up-down` input ("Throttle"). Center the stick to keep follow height constant, increase or decrease to adjust height
- _Follow Distance_ is controlled using the `forward-back` input ("Pitch"). Pushing the stick forward increases the follow distance, pulling it back decreases the distance.

```

Reply:
This explanation is much clearer! I struggled on how to explain the follow angle part!
"from user's perspective, if you face the drone ..." part is a great addition :+1: